src/main/resources/com/microsoft/jenkins/appservice/WebAppDeploymentRecorder/help-deployType.html

```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License. See License.txt in the project root for
+  ~ license information.
+  -->
+
+<div>
+    <p>The deploy type will be used.</p>
```

Comment:
this is java only, this page will be rendered only for java?

Reply:
this page will be rendered for all languages, other languages should choose Git deployment.